src/usecases/adminIsAuthenticated.js

```diff
@@ -8,7 +8,7 @@ export default ({ getTokenProvider }) => (requestCookie) => {
 
     const validatedToken = tokenProvider.validate(token);
 
-    if (token && !validatedToken.admin) {
+    if (token && validatedToken.type !== "trustAdmin") {
```

Comment:
We could define the string that corresponds to a type in a single place and then reference it when needed.

Reply:
Ye good point it would be nice to store them as constants somewhere

Reply:
Moved the token types out to a constant in 62584eee351103992